### DISCUSSION TITLE
TENSORCHORD TEA HOUR 技术分享: BUILDKIT INTERNALS | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#11] [General]
# TENSORCHORD TEA HOUR 技术分享: BUILDKIT INTERNALS | 高策

江湖小虾米

http://gaocegege.com/Blog/kubernetes/tensorchord-tea-hour-1

**Comment by geckofu** [comment]
Thanks for sharing :heart:

**Comment by wuvei** [comment]
请问大佬这个分享有视频回放吗？

**Reply by gaocegege** [reply]
https://www.youtube.com/channel/UCCA7u-PLO1fP8j1X7Pgut1Q
现在只有在 YouTube 上有

**Reply by wuvei** [reply]
谢谢大佬！
